Not accepting unions in avro schema

## Describe the issue

Im trying to convert an avro file to csv using this github which I ran directly in the cloud shell.
The build failed with this error: "An exception occured while executing the Java class. Field type union is not supported"

My avro schema indeed has unions, for example: {"default": null, "name": "tech_filename", "type": ["null", "string"]}.

I tried removing them but the dataflow job fails due to "java.io.IOException: Failed to start reading from source" which comes from "Caused by: org.apache.avro.AvroTypeException: Found null, expecting int".

So I need the unions "type":["null", "int"] to take care of these exceptions but it isn't accepted by the build.

Is there any chance to get the build to accept unions in the avro schema?

--- a/dataflow/transforms/src/main/java/com/example/AvroToCsv.java
+++ b/dataflow/transforms/src/main/java/com/example/AvroToCsv.java
@@ -38,7 +38,7 @@ public class AvroToCsv {
 
   private static final Logger LOG = LoggerFactory.getLogger(AvroToCsv.class);
   private static final List<String> acceptedTypes = Arrays.asList(
-      new String[]{"string", "boolean", "int", "long", "float", "double"});
+      new String[]{"string", "boolean", "int", "long", "float", "double", "union"});
 
   private static String getSchema(String schemaPath) throws IOException {
     ReadableByteChannel channel = FileSystems.open(FileSystems.matchNewResource(

